rclpy/rclpy/timer.py

```diff
@@ -22,6 +22,7 @@ def __init__(self, timer_handle, timer_pointer, callback, timer_period_ns):
         [self.timer_handle, self.timer_pointer] = _rclpy.rclpy_create_timer(timer_period_ns)
         self.timer_period_ns = timer_period_ns
         self.callback = callback
+        self._is_canceled = False
```

Comment:
What is this member being used for? All locations seem to always call the C API anyway.